sdk/openai/openai/sources/customizations/api/policies/formDataPolicy.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import type { FormDataEncoder } from "../../../../node_modules/form-data-encoder/@type/index.d.ts";
```

Comment:
Why did we need to import this way versus the previous way?

Reply:
Their "exports" section doesn't define types for the exported modules, you get this error otherwise:
![image](https://github.com/Azure/azure-sdk-for-js/assets/6074665/a24a86ed-b5b3-4872-ba89-33fbebaf3683)